public/data/CyberDevv.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Odesola Ibrahim",
+  "type": "personal",
+  "bio": "I am a Full-Stack Developer using React and Node.js",
+  "avatar": "https://github.com/cyberdevv.png",
```

Comment:
```suggestion
  "avatar": "https://github.com/CyberDevv.png",
```

Reply:
Pretty sure URLs are case-insensitive.

Reply:
The current link should work, but I think it's best to use its correct syntax.

Reply:
@Panquesito7 This is something which has no effect whatsoever and achieves nothing by changing it.